tests/deployment/serverless/test_aws_lambda.py

```diff
@@ -0,0 +1,194 @@
+from moto import mock_cloudformation, mock_lambda, mock_s3, mock_apigateway
+from ruamel.yaml import YAML
+
+from bentoml.deployment.serverless.aws_lambda import (
+    generate_aws_lambda_serverless_config,
+    generate_aws_lambda_handler_py,
+    AwsLambdaDeploymentOperator,
+)
+from bentoml.proto.deployment_pb2 import Deployment
+from bentoml.proto.repository_pb2 import Bento, BentoServiceMetadata, GetBentoResponse
+from bentoml.proto.status_pb2 import Status
+from bentoml.deployment.serverless import serverless_utils
+
+
+def test_generate_aws_lambda_serverless_config(tmpdir):
+    python_version = '3.7.2'
+    deployment_name = 'test_deployment_lambda'
+    api_names = ['predict']
+    region = 'us-west-test'
+    namespace = 'namespace'
+    generate_aws_lambda_serverless_config(
+        python_version, deployment_name, api_names, str(tmpdir), region, namespace
+    )
+    config_path = os.path.join(str(tmpdir), 'serverless.yml')
+    yaml = YAML()
+    with open(config_path, 'rb') as f:
+        yaml_data = yaml.load(f.read())
+    assert yaml_data['service'] == deployment_name
+    assert yaml_data['functions']['predict']['handler'] == 'handler.predict'
+    assert yaml_data['provider']['region'] == 'us-west-test'
+
+
+def test_generate_aws_lambda_handler_py(tmpdir):
+    bento_name = 'bento_name'
+    api_names = ['predict', 'second_predict']
+    generate_aws_lambda_handler_py(bento_name, api_names, str(tmpdir))
+
+    import sys
+
+    sys.modules['bento_name'] = Mock()
+
+    sys.path.insert(1, str(tmpdir))
+    from handler import predict, second_predict
+
+    assert predict.__module__ == 'handler'
+
+
+def mock_describe_cf(self, name):
+    if name == 'DescribeStacks':
+        return {
+            "Stacks": [
+                {
+                    "Outputs": [
+                        {
+                            "OutputKey": "ServiceEndpoint",
+                            "OutputValue": 'https://base_url.lambda.aws.com',
+                        }
+                    ]
+                }
+            ]
+        }
+    return
+
+
+def mock_aws_lambda_deployment_wrapper(func):
+    if version_info.major >= 3:
+        mock_open_param_value = 'builtins.open'
+    else:
+        mock_open_param_value = '__builtin__.open'
+
+    @patch('subprocess.check_output', autospec=True)
+    @patch('subprocess.check_call', autospec=True)
+    @patch.object(serverless_utils, 'check_nodejs_compatible_version', autospec=True)
+    @patch(mock_open_param_value, mock_open(), create=True)
+    @patch('shutil.copytree', autospec=True)
+    @patch('shutil.copy', autospec=True)
+    @patch.object(serverless_utils, 'init_serverless_project_dir', autospec=True)
+    @patch('subprocess.Popen')
+    @patch('botocore.client.BaseClient._make_api_call', new=mock_describe_cf)
+    def wrapper(*args, **kwargs):
+        return func(*args, **kwargs)
+
+    return wrapper
+
+
+def generate_fake_yatai_service(is_local=False):
```

Comment:
nit:  rename to `generate_yatai_service_mock`

Reply:
and about is_local parameter, see previous comment

Also since majority of the tests are using local repository, would be nice to use that as default instead of making non-local a default here